src/app/pages/resources/overview/overview.jsx

```diff
@@ -28,22 +32,43 @@ const useStyles = makeStyles({
   pageBody: {
     display: 'grid',
     gap: '16px',
-    gridTemplateColumns: '1fr 4fr',
+    gridTemplateColumns: '1fr',
     padding: '32px',
     backgroundColor: '#f9f9f9',
+    [theme.breakpoints.up('md')]: {
+      gridTemplateColumns: '1fr 4fr',
+    },
   },
   content: {
     display: 'grid',
-    gap: '32px',
+    gap: '8px',
     alignItems: 'start',
     gridTemplateColumns: '1fr',
-    [theme.breakpoints.up('md')]: {
-      gridTemplateColumns: '7fr 5fr',
-    },
+    overflowWrap: 'anywhere',
   },
   fullWidth: {
     gridColumnEnd: 'span 2',
   },
+  FabContainer: {
```

Comment:
This "case" doesn't match our other styles

Reply:
Sorry im confused on this comment, do you mean the oveflow wrap?